source/storage/apis-1.5.rst

```diff
@@ -338,26 +380,61 @@ collection.
     Posted BSOs whose ids do not appear in either "success" or "failed"
     should be treated as having failed for an unspecified reason.
 
-    Two input formats are available for multiple record POST requests,
-    selected by the *Content-Type* header of the request:
+    To allow upload of large numbers of items while ensuring that other
+    clients do not sync down inconsistent data, this API can combine several
+    POST requests into a single "batch" so that all modified BSOs appear to
+    have been submitted at the same time.  Batching behaviour is controlled
+    by the following query parameters:
 
-    - **application/json**: the input is a JSON list of objects, one for
-      for each BSO in the request.
+    - **batch**: indicates that uploads should be batched together into a
+      single conceptual update.  To begin a new batch pass the string 'true'.
+      To add more items to an existing batch pass a previously-obtained batch
+      identifier.
```

Comment:
"If the server doesn't support batching, this parameter will be ignored, and the server won't supply a batch identifier with its response."